api-tests/features/return-codes.feature

```diff
@@ -0,0 +1,174 @@
+Feature: Return exit codes
+  Rule: no return codes
```

Comment:
(Are these `Rule`s adding much? Feels like they could just be merged with the scenario names...)